hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/DFSConfigKeysLegacy.java

```diff
@@ -29,58 +29,58 @@ private DFSConfigKeysLegacy() {
   }
 
   public static final String DFS_DATANODE_DNS_INTERFACE_KEY =
-      "dfs.datanode.dns.interface";
+      "hdds.datanode.dns.interface";
   public static final String DFS_DATANODE_DNS_NAMESERVER_KEY =
-      "dfs.datanode.dns.nameserver";
+      "hdds.datanode.dns.nameserver";
 
   public static final String DFS_DATANODE_HOST_NAME_KEY =
-      "dfs.datanode.hostname";
+      "hdds.datanode.hostname";
 
   public static final String DFS_DATANODE_DATA_DIR_KEY =
-      "dfs.datanode.data.dir";
+      "hdds.datanode.data.dir";
 
   public static final String DFS_DATANODE_USE_DN_HOSTNAME =
-      "dfs.datanode.use.datanode.hostname";
+      "hdds.datanode.use.datanode.hostname";
 
   public static final boolean DFS_DATANODE_USE_DN_HOSTNAME_DEFAULT = false;
 
-  public static final String DFS_XFRAME_OPTION_ENABLED = "dfs.xframe.enabled";
+  public static final String DFS_XFRAME_OPTION_ENABLED = "hdds.xframe.enabled";
 
   public static final boolean DFS_XFRAME_OPTION_ENABLED_DEFAULT = true;
 
-  public static final String DFS_XFRAME_OPTION_VALUE = "dfs.xframe.value";
+  public static final String DFS_XFRAME_OPTION_VALUE = "hdds.xframe.value";
 
   public static final String DFS_XFRAME_OPTION_VALUE_DEFAULT = "SAMEORIGIN";
 
   public static final String DFS_METRICS_SESSION_ID_KEY =
-      "dfs.metrics.session-id";
+      "hdds.metrics.session-id";
 
   public static final String NET_TOPOLOGY_NODE_SWITCH_MAPPING_IMPL_KEY =
       "net.topology.node.switch.mapping.impl";
 
   public static final String DFS_CLIENT_HTTPS_KEYSTORE_RESOURCE_KEY =
-      "dfs.client.https.keystore.resource";
+      "hdds.client.https.keystore.resource";
 
   public static final String DFS_SERVER_HTTPS_KEYSTORE_RESOURCE_KEY =
-      "dfs.https.server.keystore.resource";
+      "hdds.https.server.keystore.resource";
 
-  public static final String DFS_HTTP_POLICY_KEY = "dfs.http.policy";
+  public static final String DFS_HTTP_POLICY_KEY = "hdds.http.policy";
 
   public static final String DFS_DATANODE_KERBEROS_PRINCIPAL_KEY =
-      "dfs.datanode.kerberos.principal";
+      "hdds.datanode.kerberos.principal";
 
   @Deprecated
   public static final String DFS_DATANODE_KEYTAB_FILE_KEY =
-      "dfs.datanode.keytab.file";
+      "hdds.datanode.keytab.file";
 
   public static final String DFS_DATANODE_KERBEROS_KEYTAB_FILE_KEY =
-      "dfs.datanode.kerberos.keytab.file";
+      "hdds.datanode.kerberos.keytab.file";
```

Comment:
`dfs.datanode.keytab.file` is already deprecated in favor of `dfs.datanode.kerberos.keytab.file`.  So we can remove `DFS_DATANODE_KEYTAB_FILE_KEY`.